Userland/Applications/HexEditor/HexEditorWidget.cpp

```diff
@@ -309,11 +309,11 @@ void HexEditorWidget::update_inspector_values(size_t position)
 
         value_inspector_model->set_parsed_value(ValueInspectorModel::ValueType::SignedByte, DeprecatedString::number(static_cast<i8>(unsigned_byte_value)));
         value_inspector_model->set_parsed_value(ValueInspectorModel::ValueType::UnsignedByte, DeprecatedString::number(unsigned_byte_value));
-        value_inspector_model->set_parsed_value(ValueInspectorModel::ValueType::ASCII, DeprecatedString::formatted("{:c}", static_cast<char>(unsigned_byte_value)));
+        value_inspector_model->set_parsed_value(ValueInspectorModel::ValueType::ASCII, make_display_string(DeprecatedString::formatted("{:c}", static_cast<char>(unsigned_byte_value))));
     } else {
         value_inspector_model->set_parsed_value(ValueInspectorModel::ValueType::SignedByte, "");
         value_inspector_model->set_parsed_value(ValueInspectorModel::ValueType::UnsignedByte, "");
-        value_inspector_model->set_parsed_value(ValueInspectorModel::ValueType::ASCII, "");
+        value_inspector_model->set_parsed_value(ValueInspectorModel::ValueType::ASCII, make_display_string(""));
```

Comment:
This seems somewhat pointless

Reply:
That's fair, I was kinda on the fence about it. I went ahead and removed them